pkg/model/proto/model.proto

```diff
@@ -735,6 +735,7 @@ message CreateFileShareOpts {
 message DeleteFileShareOpts {
     // The uuid of the fileshare, required.
     string id = 1;
+    string profileId = 2;
```

Comment:
Like I said, there is no need to add `profileId` here.

Reply:
Sorry forgot to do the same for delete, now i did